### ISSUE TITLE
Using DueDate renders XML invalid

### ISSUE THREAD

**Issue by rodehoed** [num-num/ubl-invoice#14]
When specifying a DueDate the generated XML is not valid. DueDate must sit between **IssueDate** and **InvoiceTypeCode** 
#13 fixes this.

**Comment by brtdv** [comment]
Thank you so much @rodehoed

**Comment by brtdv** [comment]
Merged, obviously!

**Comment by rodehoed** [comment]
Hi Bert! Is it possible there has been a regression on this one? As it currently does not validate anymore. This is the generated xml:
```
 <cbc:IssueDate>2020-10-02</cbc:IssueDate>
 <cbc:InvoiceTypeCode>380</cbc:InvoiceTypeCode>
 <cbc:DueDate>2020-10-16</cbc:DueDate>
```
Duedate should be between IssueDate and Invoicetype to validate against 1.3.2

**Comment by brtdv** [comment]
That's weird, I'll double check this...

**Comment by brtdv** [comment]
You were right. This should now be fixed in [v1.9.4](https://github.com/num-num/ubl-invoice/commit/a98bfd180b9c9c8386a44a415a6c3aceb2ec78c4). I've also added an [extra unit test](https://github.com/num-num/ubl-invoice/blob/master/tests/DueDateTest.php) that uses the <cbc:DueDate> element to prevent this from happening again...
